java/java.lsp.server/vscode/package.json

```diff
@@ -609,13 +594,32 @@
 				"title": "Open Service Console"
 			},
 			{
+<<<<<<< HEAD
 				"command": "nbls:Tools:org.netbeans.modules.cloud.oracle.actions.AddDbConnectionToVault",
 				"title": "Add to OCI Vault"
 			},
 			{
 				"command": "java.select.editor.projects",
+=======
+				"command": "nbls.select.editor.projects",
+>>>>>>> f6e7f94eee (Prefixing all code names of commands with 'nbls'.)
 				"title": "Reveal active editor in Projects",
 				"category": "Project"
+			},
+			{
```

Comment:
I'm pretty sure this shouldn't be here?!

Reply:
Sorry for that. I thought I've fixed all the conflicts, but I apparently didn't.

Reply:
No worries!  I think it would have been picked up in tests if the labels were correctly set.